<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-VVFUTVG" failures="0" tests="4" name="com.Doctor_module.System_TestingTest" time="16.716" errors="1" timestamp="2024-01-19T15:42:49 IST" skipped="3">
  <testcase classname="com.Doctor_module.System_TestingTest" name="Systest" time="3.030">
    <skipped/>
  </testcase> <!-- Systest -->
  <system-out>
    <![CDATA[Systest]]>
  </system-out>
  <testcase classname="com.Doctor_module.System_TestingTest" name="Systest" time="4.989">
    <skipped/>
  </testcase> <!-- Systest -->
  <system-out>
    <![CDATA[Systest]]>
  </system-out>
  <testcase classname="com.Doctor_module.System_TestingTest" name="Systest" time="3.332">
    <error message="null value in entry: url=null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: null value in entry: url=null
at com.google.common.collect.CollectPreconditions.checkEntryNotNull(CollectPreconditions.java:33)
at com.google.common.collect.SingletonImmutableBiMap.<init>(SingletonImmutableBiMap.java:43)
at com.google.common.collect.ImmutableBiMap.of(ImmutableBiMap.java:81)
at com.google.common.collect.ImmutableMap.of(ImmutableMap.java:128)
at org.openqa.selenium.remote.DriverCommand.GET(DriverCommand.java:191)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:300)
at com.Doctor_module.System_TestingTest.Systest(System_TestingTest.java:48)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </error>
  </testcase> <!-- Systest -->
  <system-out>
    <![CDATA[Systest]]>
  </system-out>
  <testcase classname="com.Doctor_module.System_TestingTest" name="Systest" time="5.365">
    <skipped/>
  </testcase> <!-- Systest -->
  <system-out>
    <![CDATA[Systest]]>
  </system-out>
</testsuite> <!-- com.Doctor_module.System_TestingTest -->
